docs/windows_installer_tutorial/AutomatedInstallerWorkflowSetup.md

```diff
@@ -0,0 +1,150 @@
+```
+and restart.
+
+### Prepare the driver
+
+Go to https://github.com/SubconsciousCompute/fsfilter-rs/releases/tag/v0.8.0 and download the "Source code" zip and extract it to this folder. To apply the patch file, run (on Linux/WSL)
+```
+patch -s -p0 < fsfilter.patch
+```
+
+## Sections for VM Setup
+### Enable shared clipboard
+
+In the top bar, Go to Devices -> Shared Clipboard and select "Host To Guest" to make it easier on yourself to run commands.
+
+### Enable Administrator account
+
+Start up the VM and let it run its first time setup. Open "terminal" as administrator and run
+```
+net user "Administrator" /active:yes
+```
+and then
+```
+net user USER password
+```
+in the prompt. In the Start Menu, sign out of "USER" and login to Administrator with "password" as the password. Once the first time setup is done, pin "Terminal" to the taskbar to make it easy on yourself for later steps. Sadly Windows has heavy visual bugs on VirtualBox, so opening apps from the taskbar avoids most of that.
+
+### Install python and pywinauto
+
+In the Microsoft Store, install the latest python. In powershell, run
+```
+python -m pip install --upgrade pip
+```
+and
+```
+pip install pywinauto
+```
+to install the libraries needed for the scripts.
+
+### Move files into the VM
+
+In the VirtualBox VM manager on the VM, select Settings -> Shared Folders, then select AutoInstallerTool and leave the "Mount point" field blank, check "Auto-mount" then hit OK. From the shared folder (probably the Z: volume), move ```fsfilter-rs-0.8.0\``` and ```setupstepper.py``` into the Documents directory (anywhere works).
+
+### Install Rust
+
+Open Edge, select "Start without your data" and "Continue without this data" and Uncheck the box and "Confirm and continue" and Uncheck the box again and "Confirm and start browsing." Finally, go to https://www.rust-lang.org/tools/install and download Rustup-init.exe as 64-bit. Disconnect from any corporate networks to avoid certificate issues from now on. Open the installer in powershell, type 1, then hit Enter. Once that's done, hit Enter and close
+the shell.
+
+### Download the EWDK
+
+Go to https://learn.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk and download the EWDK under the section that comes right before "Driver samples for Windows" and accept the license terms. This should be a few Gigabytes large.
+
+### Enable unsigned drivers
+
+Hit "Win + X" on the keyboard and select "Shut down or sign out" and then "Shift + Click" the "Restart" option. Then, Troubleshoot -> Advanced options -> Start-up Settings -> Restart -> "Disable driver signature enforcement" which can be selected by hitting "7" on the keyboard. Now you can sign out and login as Administrator again.
+
+### Build the driver and run it
+
+Right click on the downloaded EWDK and mount it. It might get mounted to the ```D:``` drive, so I'll refer to that one. Open "cmd" in the ```fsfilter-rs-0.8.0\minifilter``` directory and run

```

Comment:
```suggestion
Right click on the downloaded EWDK and mount it. It might get mounted to the `D:` drive, so I'll refer to that one. Open "cmd" in the `fsfilter-rs-0.8.0/minifilter` directory and run
```